app/views/devise/registrations/new.html.erb

```diff
@@ -53,7 +52,7 @@
   </div>
 
   <div class="field-group">
-    <%= label_tag :user_school_id, 'Your school:', :id => 'userSchool-label' %>
+    <%= label_tag :user_school_id, 'Your school', :id => 'userSchool-label' %>
```

Comment:
Using a label for id info might be problematic for accessibility, but let's write this down and move forward and do accessibility when we have Willa.